src/pages/components/breadcrumb/style.mdx

```diff
@@ -6,82 +6,194 @@ description:
 
-| Element    | Property    | px/rem  | Spacing token |
-| ---------- | ----------- | ------- | ------------- |
-| Breadcrumb | margin-left | 8 / 0.5 | `$spacing-03` |
+### Small breadcrumb structure
 
-<div className="image--fixed">
+| Element                | Property                   | px/rem        | Spacing token         |
+| -------                | ------------------         | ------------- | ------------------    |
+| Link                   | margin-left, margin-right  | 4 / 0.25      | `$spacing-02`         |
+| Link: overflow icon    | margin-left, margin-right  | 4 / 0.25      | `$spacing-02`         |
+| Link: overflow menu    | height                     | 40 / 2.5      | `$spacing-08`         |
 
-![Truncated breadcrumb dropdown example](images/breadcrumb-style-2.png)
+<Row>
+<Column colLg={8}>
 
-</div>
+![Structure and spacing measurements for small breadcrumb | px / rem](images/breadcrumb-style-structure-small-1.png)
 
-<Caption>Structure and spacing measurements for breadcrumb | px / rem</Caption>
+<Caption>Structure and spacing measurements for small breadcrumb | px / rem.</Caption>
 
-## Truncated breadcrumbs
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![Structure and spacing measurements for small breadcrumb overflow | px / rem](images/breadcrumb-style-structure-small-overflow-2.png)
+
+<Caption>Structure and spacing measurements for small breadcrumb overflow | px / rem.</Caption>
+
+</Column>
+</Row>
+
+### Medium breadcrumb structure
+
+| Element                | Property                   | px/rem        | Spacing token         |
+| -------                | ------------------         | ------------- | ------------------    |
+| Link                   | margin-left, margin-right  | 8 / 0.5       | `$spacing-03`         |
+| Link: overflow icon    | margin-left, margin-right  | 8 / 0.5       | `$spacing-03`         |
+| Link: overflow menu    | height                     | 40 / 2.5      | `$spacing-08`         |
+
+<Row>
+<Column colLg={8}>
+
+![Structure and spacing measurements for medium breadcrumb | px / rem](images/breadcrumb-style-structure-medium-1.png)
+
+<Caption>Structure and spacing measurements for medium breadcrumb | px / rem.</Caption>
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![Structure and spacing measurements for medium breadcrumb overflow | px / rem](images/breadcrumb-style-structure-medium-overflow-2.png)
+
+<Caption>Structure and spacing measurements for medium breadcrumb overflow | px / rem.</Caption>
+
+</Column>
+</Row>
+
+### Size
+
+| Element        | Size        | Height (px/rem)           |
+| ------------   | --------    | ---------------------     |
+| Breadcrumb     | Small (sm)  | 16 / 1                    |
+|                | Medium (md) | 18 / 1.125                |
+
+<Row>
+<Column colLg={8}>
+
+![Size measurements for small and medium breadcrumb | px / rem](images/breadcrumb-style-size-1.png)
```

Comment:
After seeing this Size imagw live in the website preview, you can just annotate 16px and 18px in magenta and remove the rem part in the image. For most of our Size section images on the website, we don't include the rem here, so just following this for consistency.